.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+package.json
```

Comment:
These files are important and usually don't go to ignore in real projects
`package.json` - contains all common info about the project, necessary to recreate it on the new machine.
`package-lock.json` - contains info about libs nesting and connection. It helps to recreate the exact same state of the project. Useful for production builds.